lookout/style/format/code_generator.py

```diff
@@ -185,6 +195,7 @@ def _iterate_vnodes(self, vnodes, vnodes_y, y_pred):
                 y_index += 1
 
     def _error_handler(self, msg):
+        msg += " url: %s, commit: %s." % (self.url, self.commit)
```

Comment:
both additions are mergeable into one (with format string). It will be more readable.